src/requirements/data/majors/comm.ts

```diff
@@ -61,51 +53,50 @@ const commRequirements: readonly CollegeOrMajorRequirement[] = [
       'A student may elect to fulfill 3 of these credit hours by taking a third focus area introductory course. ' +
       'A maximum of 3 credits in either 4970 or 4990 (combined) can be counted toward the upper level major requirements. Refer to the Course and Time Roster for the most up-to-date offerings. ',
     source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
-    checker: includesWithSubRequirements(
-      ['COMM 2179'],
-      ['ANTHR 4330', 'COGST 4330', 'STS 4330'],
-      ['COML 2703', 'ENG 2703', 'MUSIC 2703', 'PMA 2703'],
```

Comment:
I only kept one, because i found out this list was all the cross listed courses.